signac/core/mongodbcollection.py

```diff
@@ -0,0 +1,147 @@
+# Copyright (c) 2020 The Regents of the University of Michigan
+# All rights reserved.
+# This software is licensed under the BSD 3-Clause License.
+"""Implements Mongo-backend.
+
+This implements the Mongo-backend for SyncedCollection API by
```

Comment:
```suggestion
This implements the MongoDB-backend for SyncedCollection API by
```
"MongoDB" is a name, please make sure to use it consistently correctly.